index.html

```diff
@@ -1,138 +1,9 @@
-            margin: .5em;
-        }
-
-        .mario {
-            color: #e60012;
-            font-weight: bolder;
-            background-color: white;
-            border-radius: 18px;
-            border: .5em solid #e60012;
-            padding: .5em;
-            margin: .5em;
-            box-shadow: 0px 0px 2px 1px black;
-        }
-
-        h1 {
-            font-weight: bolder;
-            /*-webkit-text-stroke-width: 1px;
-            -webkit-text-stroke-color: black;*/
-            display: table;
-            font-size: 2em;
-            padding: 0;
-            margin: .125em;
-            line-height: 1em;
-            text-shadow: rgba(0,0,0,.05);
-            text-align: center;
-            text-shadow: 1px 1px 2px black;
-        }
-        
-        h2 {
-            color: black;
-            font-weight: 800;
-            /*-webkit-text-stroke-width: 1px;
-            -webkit-text-stroke-color: black;*/
-            padding-left: 0em;
-            text-align: center;
-            text-shadow: 1px 1px 1px black;
-        }
-
-    </style>
-</head>
-
-<body>
-    <main class="main">
-    <div class="mario">
-        <h1>MARIO GOLF SUPER RUSH</h1>
-        <h2>Team Up Bot Game Results Tool</h2>
-    </div>
-    <div class="fairway">
-       
-        <label for="playerlist">First Place</label>
-        <br>
-        <select name="playerlist" list="players" id="firstplace">
-            <option label=" "></option>
-        </select>
-            <button id="1st" onclick="AddResult(this)">Add</button>
-        <br>
-        <label for="playerlist">Second Place</label><br>
-        <select name="playerlist" list="players" id="secondplace">
-            <option label=" "></option>
-        </select>
-            <button id="2nd" onclick="AddResult(this)">Add</button>
-        <br>
-        <p style="display: none">
-        <label for="playerlist">Third Place</label><br>
-        <select name="playerlist" list="players" id="thirdplace">
-            <option label=" "></option>
-        </select>
-            <button id="3rd" onclick="AddResult(this)">Add</button>  
-        <br>
-        <label for="playerlist">Fourth Place</label><br>
-        <select name="playerlist" list="players" id="fourthplace">
-            <option label=" "></option>
-        </select>
-            <button id="4th" onclick="AddResult(this)">Add</button>
-        </p>  
-    </div>
-    <div class="fairway">
-        
-            <label for="results">Results Command</label><br>
-            <textarea id="results">/game record leaderboard: MGSR result:</textarea>
-            <br>
-        <button onclick="reset()">Reset</button>&nbsp;<button onclick="copyClipboard()">Copy to Clipboard</button>
-    </div>
-    
-    
-    </main>
-    <script src="app.js"></script>
-</body>
-
+<html>
+  <head>
+    <meta http-equiv="refresh" content="0; url='mgsr/'" />
+  </head>
+  <body>
+    <p>Updated tool available <a href="mgsr/">here</a></p>
```

Comment:
This makes the original root directory index.html of this MGSR-TeamUp folder will auto-redirect to our newer tool.  Most browsers will auto-redirect, but a few old ones may not so, just in case, include a clickable link here too.